README.md

```diff
@@ -70,6 +70,12 @@ MailerSend PHP SDK
     * [Get a list of templates](#get-a-list-of-templates)
     * [Get a single template](#get-a-single-template)
     * [Delete a template](#delete-a-template)
+  * [Email Verification API](#email-verification)
```

Comment:
"Email verification" without "API" is enough.

Reply:
I did that for consistency as we have `Email API`, `Activity API` and so forth. The whole README could be reviewed and changed but it's out of scope for this task.